README.md

```diff
@@ -28,56 +28,70 @@ familiar with this ask a dev or an admin to help you.
 
 - Once you are happy with your changes you can save by clicking on the blue "Commit changes" button in the upper right hand corner.
 Commit is the term github and git use for saving.
 
-![Screenshot of the location of the commit button on the edited page in Github](/non-info/04-help-commit-github.png)
+<kbd>![Screenshot of the location of the commit button on the edited page in Github](/non-info/04-help-commit-github.png)</kbd>
 
 - Add a brief note in the "Commit message" area summarizing your edit. You may add longer
 note in the `Extended description` area if you have more to say about your changes.
 
-- Make sure you have an email associated with your commits.
+- Make sure you have your email associated with your commits.
 
 - Create a "new branch for this request and start a pull request" will be selected
-- Name your branch after your JIRA ticket to link it. Ex `arxivce-1503-update-arxiv-docs-readme`
+- Name your branch after your JIRA ticket to link it. Ex `arxivce-1503-update-arxiv-docs-readme`.
 
-![Screenshot of how to create a new branch for the edits made in Github](/non-info/05-help-newBranch.png)
+<kbd>![Screenshot of how to create a new branch for the edits made in Github](/non-info/05-help-newBranch.png)</kbd>
 
-- **Please note:** The above step will only put your changes on develop and they will not be viewable on info.arxiv.org. At this point you have only saved the edit in github.
+- **Please note:** The above step will only put your changes on a branch on develop and they will not be viewable on info.arxiv.org. At this point you have only saved the edit in github.
 
 If you have more changes to make at this time you may make and commit them all on the branch you created. The additional changes you commit will be grouped together when you make your pull request in the next step.
 
-### 2nd: Make a Pull Request from `develop` to `master`
-Once you have completed all of your changes you need to get your changes reviewed. This is done by "creating a PR." PR stands for pull request.
+### 2nd: Make a Pull Request from your branch to `develop`
+Once you have completed all of your changes, you need to have your changes reviewed. This is done by "creating a PR." PR stands for pull request.
 
 - To make a new PR click the "Pull Requests" link in the header of the github page.
 
 - Your pull request will appear at the top of the page in a yellow box. Click on the blue button "Compare & pull request".
 
-![Screenshot how to compare changes in the pull request in github](/non-info/06-help-comparePR.png)
+<kbd>![Screenshot how to compare changes in the pull request in github](/non-info/06-help-comparePR.png)</kbd>
 
 - Add a title for your pull request and a description if the title is not self explanatory.
-- To the right, click on the gear icon to the right of "Reviewers"
+- To the right, click on the gear icon to the right of "Reviewers".
 - A dropdown will appear and you can request people to review your PR.
```

Comment:
any commentary on who should be requested to review it? (I don't know what the answer would be here)

Reply:
I'll add a note.